src/main/java/com/julienvey/trello/domain/Label.java

```diff
@@ -89,4 +116,29 @@ public Label setIdBoard(String idBoard) {
         this.idBoard = idBoard;
         return this;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof Label)) return false;
+        Label label = (Label) o;
+        return Objects.equals(id, label.id) &&
+                Objects.equals(idBoard, label.idBoard) &&
+                Objects.equals(color, label.color) &&
+                Objects.equals(name, label.name);
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
oh crap, scrolled down and saw new equals&hashcode methods :)
never mind my previous comment.
comparing by ID or using this mechanism is fine.